aziotd/src/error.rs

```diff
@@ -28,7 +25,7 @@ impl std::error::Error for ErrorKind {
         #[allow(clippy::match_same_arms)]
         match self {
             ErrorKind::GetProcessName(_) => None,
-            ErrorKind::ReadConfig(_, err) => Some(&**err),
+            ErrorKind::ReadConfig(_) => None,
```

Comment:
It would be good to preserve the source of the underlying error message instead of just discarding it.

Reply:
Fixed